cpp/src/parquet/column_reader_test.cc

```diff
@@ -644,20 +644,27 @@ TEST_F(TestPrimitiveReader, TestNonDictionaryEncodedPagesWithExposeEncoding) {
   pages_.clear();
 }
 
-class RecordReaderTest : public ::testing::Test {
+class RecordReaderPrimitiveTypeTest : public ::testing::TestWithParam<bool> {
```

Comment:
```suggestion
using ReadDenseForNullable = bool;
class RecordReaderPrimitiveTypeTest : public ::testing::TestWithParam<ReadDenseForNullable> {
```

Reply:
This may provide better readability. Otherwise one has to go all the way to the end to understand what does it mean.

Reply:
Done